chirp/drivers/ap510.py

```diff
@@ -242,7 +242,7 @@ def set_smartbeacon(self, d):
 
 class AP510Memory20141215(AP510Memory):
     """Compatible with firmware version 20141215"""
-    ATTR_MAP = dict(AP510Memory.ATTR_MAP.items() + {
+    ATTR_MAP = dict(AP510Memory.ATTR_MAP.items() | {
```

Comment:
I haven't tried this but I don't think this works as expected. Assuming `ATTR_MAP` is a dictionary:

```python
>>> ATTR_MAP = {"a": "b"}

>>> ATTR_MAP.items() | {"b":"c"}
{'b', ('a', 'b')}

>>> dict(ATTR_MAP.items() | {"b":"c"})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: dictionary update sequence element #0 has length 1; 2 is required
```